mautic/mautic
# Notify user that after enabling API they must clear cache to load new routes.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ x ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Issue(s) addressed                     | #10568 



#### Description:
While we wait for the decision on another PR that addresses the issue.
Add a note that users must clear cache after enabling API.
Please check wording.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. See the message is displayed on the API configuration page.




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11410"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
